File: vente_plus_back/factureservice/src/main/java/org/ms/facture_service/model/DevisLigne.java
package org.ms.facture_service.model;

import java.math.BigDecimal;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DevisLigne {
    private Long id;
    private Long produitId;
    private int quantite;
    private BigDecimal prixUnitaire;
}

File: vente-plus/src/app/models/devisLigne.ts
import { Produit } from './produit';

export interface DevisLigne {
    id?: number;
    produitId: number;
    quantite: number;
    prixUnitaire: number;
    totalLigne: number;
    remise: number;
    tva: number;
    produit?: Produit;
}